composer.json

```diff
@@ -11,5 +11,10 @@
         "cs:check": "php-cs-fixer fix --dry-run --diff",
         "lint": "find . -name \\*.php -not -path './vendor/*' -print0 | xargs -0 -n1 php -l",
         "test": "phpunit --configuration phpunit.unit.xml --fail-on-warning"
+    },
+    "autoload": {
+        "psr-4": {
+            "OCA\\Appointments\\":"lib/Service/Appointments"
```

Comment:
:-1: as discussed. Use the calendar namespace or you'll run into conflicts with an app with the name `appointments`